functions/cors-python/README.md

```diff
@@ -14,6 +14,26 @@ This example uses the Scaleway Serverless Framework Plugin. Please set up your e
 
 **Explanation** The function of the example is to allow most CORS requests. For more documentation on how to control access for CORS requests, see [HTTP response headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#the_http_response_headers)
 
+
+## Testing with serverless offline for Python
+
+In order to test your function locally before deployment in a serverless function, you can install our python offline testing library with:
+
+```
+pip install -r requirements-dev.txt
+```
+
+Launch your function locally:
+
+```
```

Comment:
You could add syntax highlighting for shell with `console` or `shell` or `bash`

```suggestion
```console
```